_guides/003-manage_cluster.md

```diff
@@ -42,4 +42,12 @@ sections:
         title: Compression
         description: Compression
         module: manage/features/compression
+      -
```

Comment:
I suggest moving users and policies into their own heading as it's not really storage features.

Manage storage
- Volumes
- Features
- Users
  - Policies